src/test/java/gomedic/testutil/modelbuilder/ActivityBuilder.java

```diff
@@ -27,7 +27,7 @@
      * Creates a {@code PersonBuilder} with the default details.
      */
     public ActivityBuilder() {
-        aid = new ActivityId(id++);
+        aid = new ActivityId(DEFAULT_ID);
```

Comment:
Why do you change this to non SERIAL, in this way, we cannot do build() 2 times without changing the id?

Reply:
This is for ease of testing, so that i can specify the id on my own, by using `id++`, you make the behaviour unpredictable if I change the activities in `TypicalActivities`

Reply:
got it